CryptoPrices/Sources/App/CryptoPricesApp.swift

```diff
@@ -48,5 +49,13 @@ struct CryptoPricesApp: App {
 
     init() {
         Fonts.registerAllCustomFonts()
+
+        // config showing taps and gestures on screen
+        #if DEBUG
+        ShowTime.enabled = .debugOnly
+        ShowTime.fillColor = .lightGray.withAlphaComponent(0.7)
+        ShowTime.strokeColor = .gray
+        ShowTime.disappearDelay = 0.1
```

Comment:
Optional: Just a minor improvement suggestion, maybe we can please this setup code for `ShowTime` in a extension and call it in one line here at the `init` like what we did with `Fonts` 🙏

Reply:
Updated in [9016ea9](https://github.com/nimblehq/nimble-crypto-ios/pull/49/commits/9016ea9f52212286dfea6d9037eee9543cba3be1) 👍